openedx_learning/lib/fields.py

```diff
@@ -19,7 +19,7 @@
 from .validators import validate_utc_datetime
 
 
-def create_hash_digest(data_bytes):
+def create_hash_digest(data_bytes: bytes) -> str:
```

Comment:
With a type annotation, this argument could be renamed from `data_bytes` to `data: bytes` which is much more clear (`data_bytes` could be a size in bytes, for example). But the name `data_bytes` is already used by `create_raw_content` in the public API so I'm not sure if it's OK to change.

Reply:
The only place it could be used is edx-platform, so I think it's okay to change if it's not already referenced there. I also don't think it's that important to do so though.

Reply:
Yeah honestly this is a very minor thing :p